src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -92,55 +84,10 @@ static void LogQueryImageResponse(const QueryImageResponse::DecodableType & resp
     }
 }
 
-// TODO: Add unit tests for parsing BDX URI
-CHIP_ERROR OTARequestor::ParseBdxUri(CharSpan uri, NodeId & nodeId, CharSpan & fileDesignator)
+void StartDelayTimerHandler(chip::System::Layer * systemLayer, void * appState)
```

Comment:
This is something that would benefit from being in OTARequestor driver...

Also, systemLayer argument is unused and no timer is started here. I am wondering what is happening.

Reply:
Yes, note that I'm working on separating all delayed actions into OTARequestorDriver. Should have a proposal ready tomorrow.

Reply:
Considering @Damian-Nordic is making this change, I'll leave this as is for this PR.